getting-started/dcr-guide.rst

```diff
@@ -0,0 +1,102 @@
+######################################
+Generating DCR Coadds with LSST ComCam
+######################################
+
+A walkthrough for building differential chromatic refraction (DCR) coadds by running the
+`dcrAssembleCoaddTask` on an example set of ComCam image data. Prior to generating DCR coadds, you must
+have image templates.
+
+.. note::
+
+   This guide assumes the user has access to a shared Butler repository containing data from LSST ComCam via
+   the `US Data Facility (USDF) <https://developer.lsst.io/usdf/storage.html>`__. This guide further assumes
+   the user has a recently-built version of ``lsst.distrib`` from the `LSST Science Pipelines
+   <https://developer.lsst.io/usdf/stack.html>`__ (circa ``w_2022_30`` or later).
+
+Generating AP Templates
+=======================
+For detailed instructions on generating AP Templates, please see the `Building good seeing templates
+<https://pipelines.lsst.io/getting-started/dc2-guide.html>`__ guide for details.
+
+
+Assembling DCR Coadds
+=====================
+To generate DCR coadds, you must use the `dcrAssembleCoaddTask` found in the `drp_tasks` package. This will
+require image templates. Note: You will need to register your DCR subfilters prior to creating DCR coadds by
+using the following code:
```

Comment:
Instead of using example code, I'd suggest linking to the relevant section in the Butler documentation (https://pipelines.lsst.io/modules/lsst.daf.butler/scripts/butler.html#butler-register-dcr-subfilters). This way you won't have to worry about changes, and anyone wanting to learn more has the link right there! :)